tutorials/plugins/editor/making_plugins.rst

```diff
@@ -82,6 +96,27 @@ structure:
         # Clean-up of the plugin goes here
         pass
 
+ .. code-tab:: csharp
+ 
+    #if TOOLS
+    using Godot;
+    using System;
+
+    [Tool]
+    public class NoHeroTools : EditorPlugin
```

Comment:
Shouldn't this be `CustomNode`?

Reply:
I guess the copy and paste cat sneaked in